src/api/custom-resolvers/mutations/update-loss-payee-financial-details-uk/index.test.ts

```diff
@@ -35,4 +35,16 @@ describe('custom-resolvers/update-loss-payee-financial-details-uk', () => {
       expect(success).toEqual(true);
     });
   });
+
+  describe('when an error occurs', () => {
+    it('should throw an error', async () => {
+      try {
```

Comment:
Jest execeptional handling should be written in following pattern:

```javascript
await expect(updateLossPayeeFinancialDetailsUk()).rejects.toThrow('Updating loss payee financial details UK');
```

Reply:
Great shout this is a lot better - we need to apply this in other places - added to my TODO list.

My fault for using the try/catch pattern.